code/web/src/modules/user/ProfilePicture.js

```diff
@@ -85,4 +86,4 @@ function profileState(state) {
     user: state.user,
   }
 }
-export default connect(profileState, { upload, messageShow, messageHide, updateUser })(ProfilePicture)
\ No newline at end of file
+export default connect(profileState, { upload, messageShow, messageHide, updateUser, getUser })(ProfilePicture)
```

Comment:
Is this so that we can retrieve the use again after we updated them?

Reply:
It is!! This code is from our `actions.js` file. The `connect` function here links our component `ProfilePicture` to `profileState` & all of the methods from `{ upload, ..., getUser }`- which is the way we link any component to Redux functionality.